ImperatorToCK3/Source/CK3/Character/CK3Character.cpp

```diff
@@ -0,0 +1,36 @@
+#include "CK3Character.h"
+#include "../../Imperator/Characters/Character.h"
+#include "../../Mappers/CultureMapper/CultureMapper.h"
+#include "../../Mappers/ReligionMapper/ReligionMapper.h"
+#include "Log.h"
+
+
+void CK3::Character::initializeFromImperator(
+	std::shared_ptr<ImperatorWorld::Character> impCharacter,
+	const mappers::ReligionMapper& religionMapper,
+	const mappers::CultureMapper& cultureMapper,
+	const mappers::LocalizationMapper& localizationMapper)
+{
+	ID = impCharacter->getID();
+	name = impCharacter->getName();
+	female = impCharacter->isFemale();
+	auto match = religionMapper.getCK3ReligionForImperatorReligion(impCharacter->getReligion());
+	if (match) religion = *match;
+	match = cultureMapper.cultureMatch(impCharacter->getCulture(), religion, impCharacter->getProvince(), "");
+	if (match) culture = *match;
+
+	auto impName = impCharacter->getName();
```

Comment:
don't you already save impCharacter->name to name?

Reply:
I do, thanks 😅